alcs-frontend/src/app/features/application/review/review.component.html

```diff
@@ -1,4 +1,8 @@
 <h3>Review</h3>
+<div class="notification-date">
+  <div class="subheading1">'Ready for Review' Notification Sent to Applicant</div>
```

Comment:
Same with casing

Reply:
This is a heading, so I think title case is correct.